### PR TITLE
chore: optimize login error message

### PR DESCRIPTION
**Pull request by xiaolang124**
1.Update error message when encountering silent get token error.

### CODE REVIEW

File: packages/cli/src/commonlib/codeFlowLogin.ts [hunk 1]
```diff
@@ -292,7 +292,8 @@ export class CodeFlowLogin {
           .catch(async (error) => {
             CliCodeLogInstance.necessaryLog(
               LogLevel.Error,
-              "[Login] silent acquire token : " + error.message
+              "[Login] Failed to retrieve token silently. If you encounter this problem multi times, you can try to delete `~/.fx/account` and try again. " +
```
Comment (swatDong) [inline packages/cli/src/commonlib/codeFlowLogin.ts:295]: Replace `~` with the full path?

File: packages/vscode-extension/package.nls.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
     "teamstoolkit.codeFlowLogin.loginTimeoutDescription": "Timeout waiting for login. Try again.",
     "teamstoolkit.codeFlowLogin.loginTimeoutTitle": "LoginTimeout",
     "teamstoolkit.codeFlowLogin.resultFileNotFound": "Result file not found.",
-    "teamstoolkit.codeFlowLogin.silentAcquireToken": "Failed to retrieve token silently: %s",
+    "teamstoolkit.codeFlowLogin.silentAcquireToken": "Failed to retrieve token silently. If you encounter this problem multi times, you can try to delete `~/.fx/account`, close all VSCode instances and try again. %s",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:47]: multi -> multiple

**Review by sffamily — CHANGES_REQUESTED** [review]

File: packages/vscode-extension/package.nls.json (hunk 2 above)
Comment (xiaolang124) [reply packages/vscode-extension/package.nls.json]: Done.

File: packages/cli/src/commonlib/codeFlowLogin.ts (hunk 1 above)
Comment (swatDong) [reply packages/cli/src/commonlib/codeFlowLogin.ts]: I mean, not hard-coded string, but call `os.homedir()` to get the exact value

**Review by sffamily — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

**Review by swatDong — DISMISSED** [review]

File: packages/vscode-extension/package.nls.json [hunk 3]
```diff
@@ -44,7 +44,7 @@
     "teamstoolkit.codeFlowLogin.loginTimeoutDescription": "Timeout waiting for login. Try again.",
     "teamstoolkit.codeFlowLogin.loginTimeoutTitle": "LoginTimeout",
     "teamstoolkit.codeFlowLogin.resultFileNotFound": "Result file not found.",
-    "teamstoolkit.codeFlowLogin.silentAcquireToken": "Failed to retrieve token silently: %s",
+    "teamstoolkit.codeFlowLogin.silentAcquireToken": "Failed to retrieve token silently. If you encounter this problem multiple times, you can try to delete `%s`, close all VSCode instances and try again. %s",
```
Comment (sffamily) [inline packages/vscode-extension/package.nls.json:47]: you can try to delete -> you can delete
VS Code -> Visual Studio Code

**Review by sffamily — CHANGES_REQUESTED** [review]

File: packages/vscode-extension/package.nls.json (hunk 3 above)
Comment (xiaolang124) [reply packages/vscode-extension/package.nls.json]: Done.

**Review by swatDong — APPROVED** [review]

**Review by sffamily — APPROVED** [review]

**Review by sffamily — APPROVED** [review]

**Review by LongOddCode — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Comment by swatDong** [comment]
why so many file changes... merge or rebase your working branch?

**Review by IvanJobs — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

**Pull request merged**